lwotgtelem/lwotgtelem_test.go

```diff
@@ -0,0 +1,255 @@
+package lwotgtelem
+
+import (
+	"context"
+	"crypto/tls"
+	"io"
+	"net"
+	"testing"
+	"time"
+
+	"github.com/openconfig/lemming/gnmi/gnmit"
+	"github.com/openconfig/magna/lwotg"
+	"github.com/openconfig/ygot/testutil"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials"
+
+	gpb "github.com/openconfig/gnmi/proto/gnmi"
+)
+
+const (
+	certFile = "testdata/cert.pem"
+	keyFile  = "testdata/key.pem"
+)
+
+func TestNew(t *testing.T) {
+	tests := []struct {
+		desc        string
+		inHostname  string
+		inPaths     []*gpb.Path
```

Comment:
what is the "in" short for?

Reply:
It's just the convention for data that is input to the tests `inFoo` and `wantBar` tends to be the idiomatic test naming convention.